src/table/src/table.rs

```diff
@@ -108,12 +108,7 @@ pub trait Table: Send + Sync {
     }
 
     /// Close the table.
-    async fn close(&self) -> Result<()> {
-        Ok(())
-    }
-
-    /// Drop regions
-    async fn drop_regions(&self) -> Result<()> {
```

Comment:
Why remove this function?

Reply:
After call`drop_regions` still needs to release `region` in `storage_engine`, it's better to call `drop_regions` from `mito::table::MitoTable`.